src/components/Grid.jsx

```diff
@@ -33,6 +33,22 @@ class Grid extends React.Component {
     }
   }
 
+  getLines () {
+    let lines = []
+
+    for (let i = 0; i < this.props.hours; i++) {
```

Comment:
@jirutka You'd have to create an array of numbers first and then transform it, which is less performant and (IMO) only slightly more readable option.